scripts/dockerrun.sh

```diff
@@ -115,7 +115,7 @@ execute() {
 echo "Building Docker Container using Dockerfile: $DOCKERFILE"
 
 # Get the name of Docker image.
-image=$(grep -i "^FROM " "$DOCKERFILE" | awk '{ print $2 }')
+image=$(grep -i "^FROM " "$DOCKERFILE/Dockerfile" | awk '{ print $2 }')
```

Comment:
@ravimeda - FYI if we want to pass a file then we would need to add the `-f` option and specify the `Name of the Dockerfile (Default is 'PATH/Dockerfile')`  The `-t` option is the `Name and optionally a tag in the 'name:tag' format (default [])`.  That has to remain as it is.